lib/Sema/CSSolver.cpp

```diff
@@ -1645,7 +1645,7 @@ bool ConstraintSystem::solveRec(SmallVectorImpl<Solution> &solutions,
 static bool shortCircuitDisjunctionAt(Constraint *constraint,
                                       Constraint *successfulConstraint,
                                       ASTContext &ctx) {
-
+  return false;
```

Comment:
How can this have an effect on SIL optimizer time? Are we picking generic overloads where previously we picked non-generic ones, and now we're doing more work in inlining/specialization?